node/src/neighborhood/node_record.rs

```diff
@@ -583,124 +620,100 @@ mod tests {
     #[test]
     fn node_record_partial_eq() {
         let earning_wallet = make_wallet("wallet");
+        let node_record_data = NodeRecordInputs {
+            earning_wallet: earning_wallet.clone(),
+            rate_pack: rate_pack(100),
+            accepts_connections: true,
+            routes_data: true,
+            version: 0,
+            location: None,
+        };
+        let node_record_data_duplicate = node_record_data.clone();
+        let node_record_data_with_neighbor = node_record_data.clone();
+        let node_record_data_mod_key = node_record_data.clone();
```

Comment:
If I'd been you I would never have given names to these clones. In my opinion it's only distracting. You use different names even thought they are the same... You see why you thought it could be a good idea, but I think you should've only cloned the `node_record_data` at every place where it is required. No names, simple. 

Of course, with the mutable ones it would have to work differently: there you can either keep what you have now or maybe better you could move the variable initialization as near as possible to the place where the variable is passed in the function in the next step.